src/receivers/ExpressReceiver.ts

```diff
@@ -229,31 +238,17 @@ export default class ExpressReceiver implements Receiver {
         if (stateVerification === false) {
```

Comment:
I'm wondering if we can remove this conditional now, and instead of conditionally passing the last `installUrlOptions` parameter in the case that `stateVerification` is disabled, we always pass it to `handleCallback`?

Reply:
@filmaj This logic sounds a bit weird to me too but it is an intentional one. If the stateVerification is true, the handleCallback method parses the state parameter to restore the parameter inside: https://github.com/slackapi/node-slack-sdk/blob/%40slack/oauth%402.5.0/packages/oauth/src/install-provider.ts#L497-L498

I don't think that revisiting this is scope of this PR. We are making the changes to the `/slack/install` part. The only thing I would like to apply here to remove the unnecessary `installer!` and its "eslint-disable-next-line" lines.

Reply:
OK, makes sense